transformations/teradata/index.md

```diff
@@ -32,15 +32,15 @@ INSERT INTO "transformationTable" ("id", "name") VALUES (2, 'doe');
 
 You can organize the script into [blocks](/transformations/#writing-scripts).
 
-## Bucket databases for Read-only input mapping
+## Bucket objects for Read-only input mapping
 
 Teradata organizes tables in databases. Each bucket in Keboola is represented by a separated database, which usually follow this naming pattern: `<stackPrefix>_<projectID>_<bucketID>`
-- `stackPrefix` depends on keboola stack which you use
+- `stackPrefix` - depends on keboola stack which you use
   - https://connection.keboola.com/ - `SAPI`
   - https://connection.eu-central-1.keboola.com/ - `KEBOOLA`
   - https://connection.north-europe.azure.keboola.com/ - `KEBOOLA`
```

Comment:
chybí ti tam stejně single stacky, takže by bylo lepší tam dát US je SAPI, everything else is KEBOOLA_